pkg/attest/attest_test.go

```diff
@@ -297,8 +305,9 @@ func Test_MAA(t *testing.T) {
 			keyBlob:              ProductionTestKeyBlobBytes,
 			uvmReferenceInfo:     uvm_reference_info_bin,
 
-			expectedError: errors.New("pulling maa post response failed: http response status equal to 400 Bad Request"),
-			expectErr:     true,
+			expectedError:   errors.New("http response status equal to 400 Bad Request"),

```

Comment:
Rather than keeping strings in sync everywhere, this is an example of when its useful to have typed errors that you can check.  That way the string itself can be anything.  Something like 
```go
type HTTPError string
func (h HTTPError) Error() string{
 return "http response status equal to " + string(HTTPError)
}
```

Then you could check it in the tests with `if errors.Is(HTTPError){...}` which will also deal with unwrapping the error if it gets wrapped anywhere.

Reply:
done